src/FINDOLOGIC/Export/Data/Image.php

```diff
@@ -57,6 +65,17 @@ public function getUrl()
         return $this->url;
     }
 
+    private function setUrl($url)
+    {
+        $url = DataHelper::checkForEmptyValue($url);
+
+        if (!filter_var($url, FILTER_VALIDATE_URL)) {
```

Comment:
URL validation must under all circumstances work the same via library and schema, i.e. same pattern.

Reply:
URL validation is needed in multiple places, such as product URL and image URL. Should be extracted either into a base class or a helper.

Reply:
Done!